test/test.js

```diff
@@ -1,15 +1,37 @@
 const jsts = require("../dist/jsts.min");
 
 console.log("Ensuring JSTS has been built with the correct methods");
-console.assert(jsts !== undefined);
-console.log("BufferOp");
-console.assert(jsts.BufferOp !== undefined, true);
-console.log("BufferOp is present!");
-
-console.log("GeoJSONReader");
-console.assert(jsts.GeoJSONReader !== undefined, true);
-console.log("GeoJSONReader is present");
-
-console.log("GeoJSONWriter");
-console.assert(jsts.GeoJSONWriter !== undefined, true);
-console.log("GeoJSONWriter is present");
+
+if (jsts !== undefined && typeof jsts === "object") {
```

Comment:
Without the console.asserts nothing will fail here I don't think? I.e. the exit code will be 0 rather than 1

Reply:
I'm pretty sure console.asserts just prints conditionally. Making this an "informational" test run, rather than something that passes or fails. Can change to be more like that if you want?

Reply:
How about I convert them to tape tests?

Reply:
I think that would be good if you have the time/willpower. I think my thought process was that we would in theory eventually delete this module as we managed to get buffer working without it, so I didn't do anything too involved. However I think it's here for the foreseeable so potentially worthwhile!

Reply:
Already done! Merge whenever you like.

> we managed to get buffer working without it

Is that a PR somewhere? turf-buffer is definitely still using BufferOp.